src/components/ui/menu-option/menu-option.scss

```diff
@@ -15,12 +15,14 @@ $menu-item-padding: 12px;
   --menu-option-border-width: 1px 0 0 2px;
   --menu-option-border-color: #{rgba(black, 0.06)};
   --menu-option-bg-hover: #{rgba(white, 0.3)};
+  --menu-option-text-color: #{$color-text-dark-alt};
 }
 
 .kui-theme--dark {
   --menu-option-border-width: 0 0 1px 2px;
   --menu-option-border-color: #{rgba(white, 0.06)};
```

Comment:
@tynandebold same for this file and in both themes

Reply:
Yeah, also remove whatever isn't used here. Just double check it somehow isn't being used in another file.